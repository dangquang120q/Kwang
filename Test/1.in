9
5
1211 937 6974 3786 8447 
17
6526 9494 5740 8966 6808 1184 1593 9504 7324 6081 3960 2693 2367 6283 8944 5245 7740 
13
5403 5807 5942 4360 7848 3591 6502 7977 3425 6645 3548 2612 9286 
4
2518 4264 9205 4833 
93
2055 1381 2082 7040 7867 3707 1922 1529 1906 4149 4433 6274 3395 3483 9901 6616 6468 205 4360 5180 1762 3057 1647 9533 9839 104 6697 569 19 5339 5395 504 4247 3788 5521 5477 5020 8731 8027 3804 9227 3519 1494 3460 5622 258 1750 1921 2502 3398 58 431 3678 6726 2388 3538 8970 861 2848 9183 4293 3624 9302 1427 1782 5273 8749 935 3018 807 2973 1919 442 8590 255 5150 9847 4026 4660 7361 9690 5595 9870 9516 3098 6328 6602 4839 1529 3124 7414 4052 8684 
31
4237 8043 2708 3428 8466 91 6822 366 2383 6894 9904 7125 9808 2837 2327 6070 8117 228 1596 9465 7939 9679 1275 8617 9117 8299 3936 8721 2409 4299 5431 
74
8898 715 3677 516 4763 343 8509 3974 2513 9402 1192 9714 1670 293 7741 6993 1376 6990 9518 1491 1872 2390 6578 5987 7145 6566 2385 6191 4150 1312 4443 685 8515 562 3998 262 5751 1077 4199 9525 8876 396 4122 7511 453 3595 7029 4003 1917 6322 1947 8362 710 9383 1738 8990 6283 2579 6182 3242 190 2546 9415 6591 5659 1912 7444 197 6279 7862 1807 6502 1971 8613 
85
3865 5800 2729 415 8325 4838 503 3177 9158 8147 9031 791 5129 2133 4562 2574 6836 2543 6364 5219 3311 7674 2314 1141 234 6538 3462 3430 4028 1421 714 316 4566 5021 9871 1612 5646 132 5958 5828 533 2403 551 6768 4338 1808 2300 518 1922 7691 5199 9880 7025 6360 6936 2373 8684 4833 3300 701 9975 5662 1472 6316 1313 3685 3182 2044 4519 1091 6729 1824 9446 2761 3905 4590 4887 299 2407 9359 7744 4229 9069 1193 5256 
58
9413 9135 3451 3944 470 2160 134 797 6461 8634 2839 7761 6333 2335 8694 384 3144 5586 8196 2279 9666 5377 9240 5266 6591 7808 1497 2110 6910 3250 2822 8569 3347 5330 4481 5588 6426 7536 3796 3809 7931 1253 9884 3864 7516 3214 887 4884 2872 7625 2771 8884 9235 4048 8419 3331 5678 1628 
