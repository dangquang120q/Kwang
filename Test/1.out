19207
27964
25111
18302
29618
29391
28757
29726
28963
